server/templates/testing-login.html

```diff
@@ -8,22 +8,17 @@
             Login for Testing
         </div>
         <div class="login-box-body">
-            {% for user in ['okstaff@okpy.org', 'student0@okpy.org'] %}
+            <p class="login-box-msg">Impersonate a random user</p>
```

Comment:
Wrap in a if block in the case when there are no users. 

`{% if random_user %}`